7a_temp_coop_munge/src/crosswalk_fxns.R

```diff
@@ -28,6 +29,8 @@ crosswalk_coop_dat <- function(outind = target_name, inind,
   dow2nhd$DOW <- gsub('mndow_', '', as.character(dow2nhd$MNDOW_ID))
   dow2nhd <- distinct(dow2nhd) # get rid of duplicated obs
 
+  iowa2hnd <- sc_retrieve(iowa_crosswalk) %>% readRDS(dowfile) %>% distinct()
```

Comment:
Did you mean for this variable name to be `iowa2nhd` rather than `iowa2hnd`?
And if so, should the second pipe action be `readRDS(iowa2nhd)` instead of `readRDS(dowfile)`?

Reply:
Good catch! Fixed in my updated commit